.github/workflows/cli.yml

```diff
@@ -101,6 +101,12 @@ jobs:
           ls targets/MY_CLI/lib | grep example_limits_response.rb
           ../openc3.sh cliroot rake build VERSION=1.0.3
           ../openc3.sh cliroot validate openc3-cosmos-cli-test-1.0.3.gem
+      - name: openc3.sh cli script list, run
+        shell: 'script -q -e -c "bash {0}"'
+        run: |
+          set -e
+          ./openc3.sh cliroot script list | grep "INST2" || true
```

Comment:
|| true will always succeed.   Need to have tests that could fail.

Reply:
Yep, done.